packages/gatsby-theme-store/src/components/Minicart/Drawer/Quantity.tsx

```diff
@@ -51,7 +52,7 @@ const MinicartQuantity = ({ variant: v, index }: Props) => {
   return (
     <QuantityWrapper isLoading={isLoading} variant={`${variant}.wrapper`}>
       <NumericStepper
-        disabled={isLoading}
+        disabled={isDisabled ?? isLoading}
```

Comment:
Take care with this operator, because if isDisabled is false but isLoading true, the value of the disabled property will be false.